mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/store/SimpleCachedFileStorageProvider.java

```diff
@@ -521,4 +523,13 @@ public List<String> initActiveVmAttributeValuesList() throws IOException {
         }
     }
 
+    @Override
+    public TaskExecutorRegistration getTaskExecutorFor(TaskExecutorID taskExecutorID) throws IOException {
+        return null;
```

Comment:
This `SimpleCachedFileStorageProvider` doesn't appear to do that the name implies. It looks a lot like the NoOp one.

Reply:
Thanks for pointing it out. Honestly, I was not very impressed by that implementation and thought I could fix it later for OSS. But, I think it might be a good idea to keep this as-is for the time being and revisit it at a later point.